arangod/Aql/Condition.h

```diff
@@ -166,7 +166,7 @@ class Condition {
                                     AstNode*, bool isPathCondition);
 
   /// @brief remove (now) invalid variables from the condition
-  bool removeInvalidVariables(VarSet const&);
+  bool removeInvalidVariables(VarSet const&, bool& isAllCoveredByIndex);
```

Comment:
```suggestion
  bool removeInvalidVariables(VarSet const&, bool& wasRemoves);
```
That name describes the meaning much better. As there is no index to cover something in  Condition

Reply:
But in such case need to be revert this bool value

Reply:
What about `noRemoves`?

Reply:
Done